### PR TITLE
WIP: Xamarin 5

### PR DESCRIPTION
**Pull request by sigurdal**
Update to Xamarin 5 + additional updates

### CODE REVIEW

**Comment by sigurdal** [comment]
To compile the tests, .NET Framework 4.8 must be installed: https://dotnet.microsoft.com/en-us/download/dotnet-framework/net48

File: Plugins.Import/Csv/GenericCsvParser.cs [hunk 1]
```diff
@@ -394,11 +387,13 @@ record = (IDictionary<string, object>) csv.GetRecord<dynamic>();
 
                     while (csv.Read())
                     {
-                        if (csv.Parser.FieldReader.IsBufferEmpty)
+                        //TODO: Fix this
```
Comment (aliverud) [inline Plugins.Import/Csv/GenericCsvParser.cs:390]: Is this a temporary fix, or planned fixed for this release?

**Review by aliverud — CHANGES_REQUESTED** [review]

File: Plugins.Import/Csv/GenericCsvParser.cs (hunk 1 above)
Comment (sigurdal) [reply Plugins.Import/Csv/GenericCsvParser.cs:390]: This should be tested with different datasets to see if the CsvHelper now nicely handles reading when reaching the end of the (temporary) buffer.

Comment (aliverud) [reply Plugins.Import/Csv/GenericCsvParser.cs:390]: We will include this in our internal build and test using various datasets.

**Pull request merged**